From: Caleb Connolly <caleb.connolly@linaro.org>
Date: Wed, 24 May 2023 18:51:59 +0100
Subject: arm64: dts: qcom: sdm845-shift-axolotl: enable pmi8998 haptics

Enable the PMI8998 haptics driver and specify the correct wave play rate

Signed-off-by: Caleb Connolly <caleb.connolly@linaro.org>
---
 arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts b/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts
index 9e68f01..239a789 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts
+++ b/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts
@@ -533,6 +533,12 @@ &pmi8998_charger {
 	status = "okay";
 };
 
+&pmi8998_haptics {
+	status = "okay";
+
+	qcom,wave-play-rate-us = <4255>;
+};
+
 &pm8998_resin {
 	linux,code = <KEY_VOLUMEDOWN>;
 	status = "okay";
